src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/LASSystem.java

```diff
@@ -45,7 +32,7 @@
     private CelestialSystem celestialSystem;
 
     /**
-     * Gives player inventory items on game start
+     * Gives an empty inventory to a player in the lobby to prevent fight's in the lobby.
```

Comment:
> to prevent fights in the lobby

Actually, I think we might want to make players invulnerable while in the lobby and allow fights :thinking: 

However, as what you wrote matches the current state, this is rather a topic for a follow-up PR.